src/dvr/dvr_db.c

```diff
@@ -135,6 +139,24 @@ int dvr_entry_is_upcoming(dvr_entry_t *entry)
   return state == DVR_RECORDING || state == DVR_SCHEDULED || state == DVR_NOSTATE;
 }
 
+int dvr_entry_is_duplicate(dvr_entry_t *entry)
+{
+	
+	 dvr_entry_sched_state_t state = entry->de_sched_state;
+	 
+	 if (showskipped == 1){
+	 	
+	 	return state == DVR_RECORDING || state == DVR_SCHEDULED || state == DVR_NOSTATE;
+	 	
+	 }else{
+	 	
+	 	if (_dvr_duplicate_event(entry) == 0){
+	 		return state == DVR_RECORDING || state == DVR_SCHEDULED || state == DVR_NOSTATE;
+	 	}else return 0;
+	 	
+	 }
+}
+
```

Comment:
Would it be possible to re-indent/fix whitespace to match existing code?

Reply:
I will fix it.